### PR TITLE
Practice next hit

### PR DESCRIPTION
**Pull request by henryaddison**

### CODE REVIEW

File: XVIII [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Shall I compare thee to a Wednesday?
+Thou art more lovely and more temperate:
+Rough winds do shake the darling buds that day,
```
Comment (amc00000) [inline XVIII]: did

File: XVIII [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Shall I compare thee to a Wednesday?
```
Comment (ghost) [inline XVIII:1]: Wednesdays are shit. So, no thanks.

Comment (henryaddison) [inline XVIII:1]: Good point. What would be a better comparison? Perhaps a season?

File: XVIII [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+And mid-week's lease hath all too short a date:
+Sometime too hot the eye of heaven shines,
+And often is his gold complexion dimmed,
+And every fair from fair sometime declines,
+By chance, or nature's changing course untrimmed:
+But thy eternal summer shall not fade,
+Nor lose possession of that fair thou ow'st,
+Nor shall death brag thou wander'st in his shade,
+When in eternal lines to time thou grow'st,
+So long as men can breathe, or eyes can see,
```
Comment (ghost) [inline XVIII]: Why does it have to be men all the time?

Comment (ghost) [inline XVIII]: I'm just saying, every one of us, man or woman, brother or sister....

File: XVIII [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Shall I compare thee to a Wednesday?
+Thou art more lovely and more temperate:
+Rough winds do shake the darling buds that day,
+And mid-week's lease hath all too short a date:
+Sometime too hot the eye of heaven shines,
+And often is his gold complexion dimmed,
```
Comment (ghost) [inline XVIII:6]: Is this about you? Blonde vanity.

File: XVIII (hunk 1 above)
Comment (ghost) [inline XVIII:3]: Shake me roughly, baby.

Comment (ghost) [inline XVIII:3]: Shake me like a British nanny.

File: XVIII [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+And every fair from fair sometime declines,
+By chance, or nature's changing course untrimmed:
+But thy eternal summer shall not fade,
+Nor lose possession of that fair thou ow'st,
+Nor shall death brag thou wander'st in his shade,
+When in eternal lines to time thou grow'st,
+So long as men can breathe, or eyes can see,
+So long lives this, and this gives life to thee.
```
Comment (amc00000) [inline XVIII]: Bring me Harry Potter.

**Pull request merged**